game-core/src/test/java/org/triplea/common/config/product/ProductConfigurationIntegrationTest.java

```diff
@@ -0,0 +1,19 @@
+package org.triplea.common.config.product;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.matchesPattern;
+
+import org.junit.jupiter.api.Test;
+import org.triplea.test.common.Integration;
+
+@Integration
+final class ProductConfigurationIntegrationTest {
+  private final ProductConfiguration productConfiguration = new ProductConfiguration();
+
+  @Test
+  void shouldReadPropertiesFromResource() {
+    assertThat(
+        productConfiguration.getVersion().getExactVersion(),
+        matchesPattern("1\\.9\\.0\\.0\\.(@qualifier@|dev|\\d+)"));
```

Comment:
The three possible qualifiers are specified here to handle the three ways the tests could be run (from the IDE, from Gradle without `productVersionQualifier` set, and from Gradle with `productVersionQualifier` set).